docs/source/roadmap.md

```diff
@@ -147,23 +76,20 @@ Nevertheless, to demonstrate _starfish_'s scale, we will implement workflows on
 ### A Developer Focus Point
 Two key aims of _starfish_ and SpaceTx are to (1) encourage convergence of the community onto a single object model for processing of image-based transcriptomics data and (2) make it easier to develop pipelines by providing a high-performance platform. To facilitate this, we must encourage developers to work on _starfish_ through active outreach and by being encouraging and helpful.
 
-## Outstanding work to complete 0.1.0
+## Near-term goals
 
 ### A General Format and Object Model for Image-based Transcriptomics
 
-#### Format specification for IntensityTable and ExpressionMatrix:
-These two objects currently lack format specifications, which has confused some users, including Ken Harris. A specification for these formats will help computational users to understand how to consume our data formats, and will help facilitate their conversion for use in other languages. For users who are comfortable with python but not necessarily with xarray, we will build in a series of `to_array` and `to_pandas` options.
+#### Format specification for _starfish_ outputs, IntensityTable and ExpressionMatrix:
+These two objects currently lack format specifications, which has confused some users. A specification for these formats will help computational users to understand how to consume our data formats, and will help facilitate their conversion for use in other languages. For users who are comfortable with python but not necessarily with xarray, we will build in a series of `to_array` and `to_pandas` options.
 
 #### _starfish_ must enable processing of fields of view by round
```

Comment:
After discussions with @ttung I'm coming to believe that this section conflates two problems: 

1. Starfish must support workflows for which data acquisition is _complete_ after each round (smFISH) and who therefore want to process individual rounds. 
2. Starfish must support the processing of single, large fields of view. For smFISH workflows, per-round processing is a good solution. For coded workflows where all rounds must be in memory simultaneously for decoding (MExFISH, SeqFISH), it may be necessary to construct a cropping framework to break up the fields of view into smaller, more manageable pieces. This now feels more like it belongs in the _pipeline runner_ category, since the problem here is making sure all the pieces of the data are processed properly. 

The key deliverables here are: 
1. Starfish can process the data
2. Starfish provides workflows that match the image acquisition patterns.

Reply:
This is nice clarification. I'm inclined to address these issues in your branch after this gets merged.